dom-events/buttons.ts

```diff
@@ -0,0 +1,29 @@
+const $clickButton = document.querySelector('.click-button');
+if (!$clickButton) throw new Error('$clickButton does not exist');
+function handleClick(event: Event): any {
+  console.log('button clicked');
+  console.log('event', event);
+  console.log('event.target', event.target);
+}
```

Comment:
You should be avoiding using `any` at all costs. If the function doesn't return anything, then what is the proper return type?